### ISSUE TITLE
[HOLD for payment 2023-10-20] [$500] Close account - There is no notification that account successfully closed

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#28748]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open the App
2. Login with any account
3. Go to Settings -> Security -> Close account
4. Enter your default contact method -> Close account -> Confirm

## Expected Result:
The notification that account successfully closed appears

## Actual Result:
There is no notification that account successfully closed

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** v1.3.77-2

**Reproducible in staging?:** Y

**Reproducible in production?:** N

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/115492554/6ec8f3ed-0267-4736-903f-3def4e4a9d91

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause-Internal Team

**Slack conversation:** @

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01379be296b685c84c</li>
        <li>Upwork Job ID: 1709308267924353024</li>
        <li>Last Price Increase: 2023-10-03</li>
<li>Automatic offers: </li>
<ul>
<li>abdulrahuman5196 | Reviewer | 27016138</li>
<li>tienifr | Contributor | 27016141</li>
</ul></li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
Not repro in Prod

https://github.com/Expensify/App/assets/115492554/c5959b7f-ffd8-4da6-8250-45e55049a5ec

**Comment by abdulrahuman5196** [comment]
Taking a look into this

**Comment by abdulrahuman5196** [comment]
@tienifr 's proposal here https://github.com/Expensify/App/issues/28748#issuecomment-1746091428 looks good and works well.
We should add the suggested fix along with the catch mentioned to make sure we see errors in console.

🎀 👀 🎀
C+ Reviewed

cc: @puneetlath

**Comment by mountiny** [comment]
CPing the fix

**Comment by kbecciv** [comment]
@mountiny  "Account closed successfully" message is displayed with delay after close it, build 1.3.77.5

https://github.com/Expensify/App/assets/93399543/b7db8569-5061-4613-a388-a615d874f688

**Comment by mountiny** [comment]
Thats probably fine

**Comment by mountiny** [comment]
We will have to pay @abdulrahuman5196 and @tienifr here

**Comment by abdulrahuman5196** [comment]
@mountiny Not sure why this issue is closed. Payment is still pending on this issue.

**Comment by abdulrahuman5196** [comment]
> Based on my calculations, the pull request did not get merged within 3 working days of assignment. Please, check out my computations here:
> 
>     * when @tienifr got assigned: 2023-10-04 10:57:56 Z
> 
>     * when the PR got merged: 2023-10-10 14:01:15 UTC
> 
>     * days elapsed: 4
> 
> 
> On to the next one 🚀

Melvin is wrong here. there was a followup PR.

**Comment by mountiny** [comment]
Seems like commit message closed this

**Comment by mountiny** [comment]
@tienifr @abdulrahuman5196 Can you please complete the checklist and leave a summary of what should be the payment here, thanks!

**Comment by abdulrahuman5196** [comment]
>The PR that introduced the bug has been identified. Link to the PR:

https://github.com/Expensify/App/pull/27996

> The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:

https://github.com/Expensify/App/pull/27996/files#r1369099302

>Determine if we should create a regression test for this bug.
If we decide to create a regression test for the bug, please propose the regression test steps to ensure the same bug will not reach production again.

No. We seem to have UTs for this.

**Comment by stephanieelliott** [comment]
**Summarizing payment on this issue:** 

 - Issue reporter: N/A
 - Contributor: @tienifr $750, paid via Upwork 
 - Contributor+: @abdulrahuman5196 $750, paid via Upwork 
 
 Upwork job is [here](https://www.upwork.com/ab/applicants/1709308267924353024/job-details), 50% bonus is included on final payout
